src/app.py

```diff
@@ -61,14 +77,15 @@ def _update_topology_repo(self):
                 app.logger.debug("topology repo update ok")
             else:
                 app.logger.warning("topology repo update failed")
```

Comment:
Can we include the git output in case of failure?

Reply:
`run_git_cmd` already logs it.